applications/sdp/gpio/src/backend/backend.h

```diff
@@ -9,7 +9,8 @@
 
 #include <drivers/gpio/nrfe_gpio.h>
 
-#if !defined(CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICMSG) && !defined(CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX)
+#if !IS_ENABLED(CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICMSG) &&                                           \
+	!IS_ENABLED(CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX)
```

Comment:
This is already modified in icbmsg PR: https://github.com/nrfconnect/sdk-nrf/pull/17321/files#diff-7fffc679701a822b8559ff5a9d885be0e635b124f4615690170ba7c39c1b177eR12-R14